docs/website/docs/v1.x/index.md

```diff
@@ -17,7 +17,7 @@ Before we dive in, make sure that you have the following on your computer:
 To use zio-http, add the following dependencies in your project:
 
 ```scala
-val ZHTTPVersion = "1.0.0.0-RC24"
+val ZHTTPVersion = "1.0.0.0-RC27"
```

Comment:
Can we not auto-generate and insert it here?

Reply:
Created a issue here
#1215